src/assets/toolpanel/Page 1.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="10" viewBox="0 0 16 10" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should the spaces in the filenames be removed? Did you mean to commit the 2 extra files in toolpanel?

Reply:
Didn't mean to add this, will remove, thanks!